Private/Remove-Subscription.ps1

```diff
@@ -6,14 +6,13 @@ function Remove-Subscription {
     )
     $Subscriptions = Start-MyProgram -Program $Script:ProgramWecutil -cmdArgList 'es'
     foreach ($Subscription in $Subscriptions) {
-        if ($Own -eq $true -and $Subscription -like '*PSWinReporting*') {
-            Write-Color 'Deleting own providers - ', $Subscription -Color White, Green
+        if ($Own -and $Subscription -like '*PSWinReporting*') {
+            $Logger.AddRecord("Deleting own providers: $Subscription")
```

Comment:
Same as few comments above. Logger is not defined for that example that uses this. Requires modification of example